returnn/search.py

```diff
@@ -185,6 +191,66 @@ def run(self):
             out.write("}\n")
 
 
+class SearchWordsToCTM(Job):
+    """
+    Convert RETURNN search output file into CTM format file
+    """
+
+    def __init__(self, recog_words_file, corpus, filter_tags=True):
+        """
+        :param Path recog_words_file: search output file from RETURNN
+        :param Path corpus: bliss xml corpus
+        :param bool filter_tags: if set to True, tags such as [noise] will be filtered out
+        """
+        self.recog_words_file = recog_words_file
+        self.corpus = corpus
+        self.filter_tags = filter_tags
+
+        self.out_ctm_file = self.output_path("search.ctm")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        corpus = Corpus()
+        corpus.load(self.corpus.get_path())
```

Comment:
corpus should be of type `tk.Path`. With `tk.uncached_path` this means you are saying that this can be a `str`. But this should not be a `str`. So `.get_path()` should be used.

Reply:
I opened an issue to talk about deprecation, but `get_path` is not wrong in any case so this is fine.